readthedocs/search/tests/conftest.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from django_dynamic_fixture import G
+from faker import Faker
+
+from readthedocs.projects.models import Project
+from readthedocs.search.indexes import Index, ProjectIndex, PageIndex, SectionIndex
+from .dummy_data import DUMMY_PAGE_JSON, ALL_PROJECTS
+
+fake = Faker()
+
+
+@pytest.fixture(autouse=True)
+def mock_elastic_index(mocker):
+    mocker.patch.object(Index, '_index', fake.word().lower())
```

Comment:
Do we need a third party package here? Could we just be generating a random index name? Or should the index name be the same for each test run perhaps?

Reply:
I think we can include it when needed. I don't want to continue to add dependencies for only 1 usage.